### PR TITLE
fix Dockerfile: dont fetch source code

### PR DESCRIPTION
**Pull request by moeryomenko**
Just copy from repo.

Signed-off-by: Maxim Eryomenko <moeryomenko@gmail.com>

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.git
+yarn.lock
```
Comment (EremenkoVO) [inline .dockerignore]: yarn.lock is need

Comment (moeryomenko) [reply .dockerignore]: for what?

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
-
 RUN npm install --global http-server
 
 WORKDIR src
 
-RUN git clone https://github.com/EremenkoVO/side-scroll-shooter.git . \
-    && yarn install \
+COPY . .
+
+RUN yarn install \
```
Comment (EremenkoVO) [inline Dockerfile]: just 'yarn'

File: .dockerignore (hunk 1 above)
Comment (EremenkoVO) [reply .dockerignore]: for true tree dependencies

Comment (moeryomenko) [reply .dockerignore]: think this no matter

**Pull request closed without merging**